crates/redstone/src/lib.rs

```diff
@@ -18,7 +18,7 @@ mod crypto;
 pub mod network;
 mod protocol;
 mod types;
-pub mod utils;
```

Comment:
This should be fine - at the same time this allows to get rid of some pub(crate) in the utils mod.rs file :D